python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -1613,6 +1615,11 @@ def getTargetList(parsedCmd, **kwargs):
                 refList1, refList2 in zip(parsedCmd.id.refList, idParser.refList)]
 
 
+def patch2Int(patch):
```

Comment:
I believe the use of `2` in the name like this is discouraged; use `To`.

Add docstring.